specificatie/zoek-personen.yaml

```diff
@@ -116,7 +116,13 @@ components:
         type:
           type: string
         fields:
-          $ref: 'filter.yaml#/components/schemas/Fields'
+          description: |
+            Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een door komma's gescheiden lijst van property namen of - paden op te geven. [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/fields_extensie.feature)
```

Comment:
Hier wordt gesproken over een door komma's gescheiden lijst. Dit zou het geval zijn als fields een query string parameter was geweest. Hier is het gewoon een lijst

Reply:
Ok, kan je je vinden in het volgende alternatief:

> Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een lijst van property paths ([zie functionele specificaties 'fields' properties](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/features/fields.feature)) of shortcuts op te geven. De te gebruiken shortcuts zijn beschreven in [fields-mapping-Persoon.csv](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/features/fields-mapping-Persoon.csv) en [fields-mapping-PersoonBeperkt.csv](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/develop/features/fields-mapping-PersoonBeperkt.csv) waarbij in de eerste kolom de shortcut staat en in de tweede kolom het met de shortcut gerelateerde property path.

Waarbij we de link hierin natuurlijk nog moeten corrigeren zodra we de 2.0.0 versie naar de master mergen.

Is hiermee het verschijnsel shortcut voldoende beschreven?